pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3862,19 +3862,21 @@ func Test_server_StreamInsert(t *testing.T) {
 
 func Test_server_MultiInsert(t *testing.T) {
 	t.Parallel()
+
 	type args struct {
 		ctx  context.Context
 		reqs *payload.Insert_MultiRequest
 	}
 	type fields struct {
 		name              string
 		ip                string
-		ngt               service.NGT
-		eg                errgroup.Group
 		streamConcurrency int
+		svcCfg            *config.NGT
+		svcOpts           []service.Option
 	}
 	type want struct {
 		wantRes *payload.Object_Locations
+		haveErr bool // skip checking error detail
```

Comment:
It is because in some test cases, the error outputted is not fixed. 
The implementation of insert function will convert the requests to the map, and then insert into ngt.
https://github.com/vdaas/vald/blob/39c94dfdfc96d6559845ef6ed14cac097aa338fc/pkg/agent/core/ngt/service/ngt.go#L707
but the order of looping through the map is randomized, so I cannot predict the output error hence adding this flag to skip checking the error detail.

Reply:
How about disassembling the error and sorting it by any new function?
Even output is not fixed, we can check it.

Reply:
instead of defining haveErr here and sort it, I implemented a for loop to check if the output error is one of the excepted error. 

Changes:
https://github.com/vdaas/vald/pull/1612/commits/7830c38f13793444902ad193655167de6bd99760